mhs-reference-implementation/mhs/routing/sds.py

```diff
@@ -56,62 +44,70 @@ def __init__(self, ldap_address: str = None, tls: bool = False):
         accredited_system_lookup = await self._accredited_system_lookup(ods_code, interaction_id)
 
         if not accredited_system_lookup:
-            raise re.RoutingException('No response from accredited system lookup')
+            raise routing_exception.RoutingException('No response from accredited system lookup')
```

Comment:
I think we need an error level log message here.

Reply:
Added this, I've also added empty response checking for the `_mhs_details_lookup` response with appropriate logging